src/Nav.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+
+class Nav extends Component {
+  constructor() {
+    super()
+  }
+
+  addFood = () => {
+    console.log('FOOD')
```

Comment:
@aprildagonese do you have any suggestions on where to start with building out the functionality for this? I'm thinking I'll need a modal with forms in order to get the user's input on name & calories.

Although I also think that by having this in the nav it means that if we want a food to be associated with a meal that we'd need the date and name of that meal. If it's in the MealFood component then both of those are taken care of already. But that could also limit when a user is adding a food... 🤷‍♂